src/description.cpp

```diff
@@ -690,6 +690,17 @@ Description::Media Description::Media::reciprocate() const {
 		break;
 	}
 
+	// Clear all ssrc attributes as them are individual
+	auto it = mAttributes.begin();
+	while (it != mAttributes.end()) {
+		if (match_prefix(*it, "ssrc:"))
+			it = reciprocated.mAttributes.erase(it);
```

Comment:
This will probably crash as `it` is an iterator on `mAttributes`, not on `reciprocated.mAttributes`.

Reply:
Sorry, I made a stupid mistake.

Reply:
It happens :smile: